### PR TITLE
fix: List app keys filter by tigris project

### PR DESCRIPTION
**Pull request by JigarJoshi**
Appkeys are associated with Tigris project. Historically - app keys were not associated with Tigris project. This PR accepts a filter while listing app keys that takes a tigris project name. It filters the app key if the tigris project name is either empty or matches the value supplied.

### CODE REVIEW

File: api/admin.go [hunk 1]
```diff
@@ -69,8 +69,9 @@ func (a *API) adminUsers(w http.ResponseWriter, r *http.Request) error {
 	filter := r.URL.Query().Get("filter")
 	namespaceFilter := r.URL.Query().Get("tigris_namespace")
 	createdByFilter := r.URL.Query().Get("created_by")
+	projectFilter := r.URL.Query().Get("tigris_project")
 
-	users, err := models.FindUsersInAudience(ctx, a.db, instanceID, aud, pageParams, sortParams, filter, namespaceFilter, createdByFilter, a.encrypter)
+	users, err := models.FindUsersInAudience(ctx, a.db, instanceID, aud, pageParams, sortParams, filter, namespaceFilter, createdByFilter, projectFilter, a.encrypter)
```
Comment (rbarabas) [inline api/admin.go:74]: This method is becoming a bit unwieldy. Should a generic filter object be better than 4 different use cases?

Comment (JigarJoshi) [reply api/admin.go:74]: sure I will refactor it when we add next filter.

Comment (rbarabas) [reply api/admin.go:74]: OK, that's fair. Let's capture as an issue, otherwise LGTM!

**Review by rbarabas — APPROVED** [review]
LGTM , adds another filter to FindUsersInAudience and we will capture follow up action item to make the filter interface a bit more generic.

File: api/admin.go (hunk 1 above)
Comment (JigarJoshi) [reply api/admin.go:74]: https://github.com/tigrisdata/gotrue/issues/23

**Comment by JigarJoshi** [comment]
> LGTM , adds another filter to FindUsersInAudience and we will capture follow up action item to make the filter interface a bit more generic.

https://github.com/tigrisdata/gotrue/issues/23 Thanks

**Pull request merged**